korman/exporter/physics.py

```diff
@@ -157,7 +157,7 @@ def generate_physical(self, bo, so, **kwargs):
 
             # Got subworlds?
             subworld = bo.plasma_object.subworld
-            if self.is_dedicated_subworld(subworld, sanity_check=False):
+            if subworld is not None and self.is_dedicated_subworld(subworld, sanity_check=False):
```

Comment:
It looks like `is_dedicated_subworld()` already returns `False` for subworlds that are `None`.